lib/efolder/service.rb

```diff
@@ -56,9 +56,13 @@ def bgs_doc_uuids
           uuids << uploaded_docs[:uuid_txt]
         else
           uploaded_docs&.each do |doc|
-            uuids << doc[:uuid_txt] if doc.is_a?(Hash) && doc.key?(:uuid_txt)
+            if doc.is_a?(Hash) && doc.key?(:uuid_txt) # rubocop:disable Style/IfUnlessModifier
+              uuids << doc[:uuid_txt]
+            end
```

Comment:
`# rubocop:disable Style/IfUnlessModifier` was added to ensure the uuid is not added to the uuids array when there are hundreds of records.